### DISCUSSION TITLE
Is there a docker image for kong-manager?

### DISCUSSION THREAD

**Discussion by LynkMonster** [Kong/kong-manager#97] [Q&A]
Seems that there's no dockerfile in this repo, a docker image will be provided in the future? thx.

**Comment by sumimakito** [comment]
Hello @LynkMonster - Kong Manager does not have a standalone Docker repo. Instead, it is bundled with Kong Gateway. You may find all Docker images for Kong Gateway CE here: https://hub.docker.com/_/kong/
